### DISCUSSION TITLE
Export excel questions

### DISCUSSION THREAD

**Discussion by MusekininKanchou** [dfinke/ImportExcel#1343] [Q&A]
I have a column with fairly long descriptions. I'd like to limit how wide that column gets when I autosize and enable wrap text on the column.  Can that be done at export or do I have to import the file and do a bunch of screwing around with it to do that?

**Comment by dfinke** [comment]
## Try this

```powershell
$xlfile = ".\spike.xlsx"
Remove-Item $xlfile -ErrorAction SilentlyContinue

$data = ConvertFrom-Csv @"
Description
This is a very long description that will be wrapped to the next line
"@  

$xl = $data | Export-Excel $xlfile -PassThru

Set-ExcelRange -Worksheet $xl.Sheet1 -Range 'A2' -WrapText -Width 12

Close-ExcelPackage $xl -Show 
```

![image](https://user-images.githubusercontent.com/67258/207469318-524ecf1a-e1bc-4121-990d-ba44e99ceb85.png)
